libs/interpret-text/src/lib/TextExplanationDashboard/Control/TextExplanationView/TextExplanationView.tsx

```diff
@@ -204,54 +213,21 @@ export class TextExplanationView extends React.PureComponent<
     this.setState({ singleTokenImportances });
   }
 
-  private calculateTopKImportances(importances: number[]): number {
-    return Math.min(
-      MaxImportantWords,
-      Math.ceil(Utils.countNonzeros(importances) / 2)
-    );
-  }
-
-  private calculateMaxKImportances(importances: number[]): number {
-    return Math.min(
-      MaxImportantWords,
-      Math.ceil(Utils.countNonzeros(importances))
-    );
+  private getImportanceForSingleToken(index: number): number[] {
+    return this.state.qaRadio === QAExplanationType.Start
```

Comment:
For cleaner code, what if we do something like-
```typescript
return this.props.dataSummary.localExplanations[+(this.state.qaRadio === QAExplanationType.Start)].map((row) => row[index])
```
This may be cleaner than repeating most of the return clause twice, and the evaluated boolean can be directly casted to an integer.

Reply:
Can be emulated in similar return clauses below as well

Reply:
thank you, refined to use variable.